jupyterlab_caip_optimizer/src/store/studies.ts

```diff
@@ -18,7 +18,12 @@ export const fetchStudies = createAsyncThunk<
     console.error('No Metadata found.');
     throw new TypeError('No metadata');
   }
-  return optimizer.listStudy(metadata);
+  const paritalStudies = await optimizer.listStudy(metadata);
```

Comment:
`partialStudies` refer to the studies without the studyconfigs, right? Could we change `partialStudies` to `studyList` and `fullStudies` to `studyListWithDetails` (or something like that)? Right now it seems like `partialStudies` is a subset of `fullStudies` just by the name (and I think `partial` is misspelled)

Reply:
> partialStudies refer to the studies without the studyconfigs, right?

You are correct.

> Could we change `partialStudies` to `studyList` and `fullStudies` to `studyListWithDetails` (or something like that)?

Yeah I can do that. This would just be a name change right, no changes to `types.ts`?

Reply:
yeah that's correct - let's try to keep `types.ts` to be exactly consistent with the Optimizer API